widgets/tabs/styles/default.less

```diff
@@ -116,19 +115,35 @@
 		}
 	}
 
-	& when ( @tabs_container_tabs_layout = vertical ) {
+	& when ( @tabs_container_tabs_placement = bottom ) {
 		& {
 			display: flex;
-			& when ( @tabs_container_tabs_placement = right ) {
-				flex-direction: row-reverse;
-			}
-			.sow-tabs-panel-container {
-				flex: 1;
-			}
+		    flex-direction: column-reverse;
+		}
+	}
 
-			.sow-tabs-tab {
-				display: block !important;
-			}
+	& when ( @tabs_container_tabs_placement = left ) {
+		display: flex;
+		
+		.sow-tabs-panel-container {
+			flex: 1;
+		}
+
+		.sow-tabs-tab {
+			display: block !important;
+		}
+	}
+
+	& when ( @tabs_container_tabs_placement = right ) {
+		display: flex;
+		flex-direction: row-reverse;
+
+		.sow-tabs-panel-container {
+			flex: 1;
+		}
+
+		.sow-tabs-tab {
+			display: block !important;
 		}
```

Comment:
Some unnecessary duplication which can be removed. I.e. the only difference between `left` and `right` is `flex-direction: row-reverse;`

Reply:
Oddly, when I initially tried this it didn't work but I just tried it again and it did. 🤷‍♂️